src/DataCollection.UWP/MainPage.xaml.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Esri.ArcGISRuntime.ExampleApps.DataCollection.Shared.Properties;
+using Esri.ArcGISRuntime.ExampleApps.DataCollection.Shared.ViewModels;
+using Windows.UI.Xaml.Controls;
+
+namespace Esri.ArcGISRuntime.ExampleApps.DataCollection.UWP
+{
+    /// <summary>
+    /// A map page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+	{
+		public MainPage()
```

Comment:
Looks like this is using tabs instead of spaces

Reply:
Microsoft did it 😆